src/components/QueryEditor/types.ts

```diff
@@ -44,14 +48,19 @@ export interface SQLOptions extends SQLConnectionLimits, DataSourceJsonData {
   timeInterval: string;
 }
 
+// Match the Enums Expected in SqlUtil and SqlDS
+// https://github.com/grafana/grafana-plugin-sdk-go/blob/main/data/sqlutil/query.go#L18-L29
 export enum QueryFormat {
-  Timeseries = 'time_series',
-  Table = 'table',
+  Timeseries,
+  Table,
+  Logs,
+  Trace,
+  OptionMulti,
 }
 
```

Comment:
This is the actual change, the rest is probably prettier or eslint doing its thing

Reply:
Is this a breaking change? Is query format saved in the query object?

Reply:
It is a breaking change. Added the changelog and version bump